packages/code-snippet-block/src/CodeSnippetBlock.tsx

```diff
@@ -4,51 +4,67 @@ import 'tailwindcss/tailwind.css';
 
 import { FC, ReactElement } from 'react';
 import { debounce } from '@frontify/fondue';
-import { useBlockSettings } from '@frontify/app-bridge';
+import { useBlockSettings, useEditorState } from '@frontify/app-bridge';
 import { toRgbaString } from '@frontify/guideline-blocks-shared';
 
 import { getValueInPx } from './helpers';
 import { CodeMirrorEditor } from './components';
-import { CodeMirrorEditorProps, CodeSnippetProps, Settings } from './types';
+import { BorderRadiusCorners, CodeMirrorEditorProps, CodeSnippetProps, Settings } from './types';
 import { BORDER_COLOR_DEFAULT_VALUE, DEFAULT_THEME_VALUE } from './constants';
 
 export const CodeSnippetBlock: FC<CodeSnippetProps> = ({ appBridge }): ReactElement => {
     const [blockSettings, setBlockSettings] = useBlockSettings<Settings>(appBridge);
+    const isEditing = useEditorState(appBridge);
 
     const {
         content,
-        padding,
+        margin,
         language,
         lineStyle,
         lineWidth,
         borderColor = BORDER_COLOR_DEFAULT_VALUE,
-        paddingTop = '0px',
-        paddingLeft = '0px',
-        paddingRight = '0px',
-        paddingBottom = '0px',
+        borderRadius = '0px',
+        marginTop = '0px',
+        marginLeft = '0px',
+        marginRight = '0px',
+        marginBottom = '0px',
+        borderRadiusTopLeft = '0px',
+        borderRadiusTopRight = '0px',
+        borderRadiusBottomLeft = '0px',
+        borderRadiusBottomRight = '0px',
         withBorder = false,
         withHeading = false,
         withRowNumbers = false,
-        withCustomPadding = false,
+        withCustomMargin = false,
+        withCustomBorderRadius = false,
         theme = DEFAULT_THEME_VALUE,
     } = blockSettings;
 
-    const customPadding = `${getValueInPx(paddingTop)} ${getValueInPx(paddingRight)} ${getValueInPx(
-        paddingBottom
-    )} ${getValueInPx(paddingLeft)}`;
+    const customMargin = `${getValueInPx(marginTop)} ${getValueInPx(marginRight)} ${getValueInPx(
```

Comment:
`getValueInPx` can be removed now, because you will get the values as pixel values from the settings (`appendUnit` helper in settings).